### ISSUE TITLE
A11y: noninteractive element cannot have positive tabIndex value

### ISSUE THREAD

**Issue by archiewood** [Mitcheljager/svelte-tiny-linked-charts#2]
Using the tiny charts like this:

```
import { LinkedChart } from "svelte-tiny-linked-charts"

let sparklineDate ={...}

<LinkedChart 
    data = {sparklineData}
    type = line
    grow = {true}
    barMinWidth = 0
    gap = 0
    fill = var(--grey-400)
    align = left
    hover = {false}
    linked = 'id'
    width = 75
    tabindex = -1
/>
```

Getting the above warning message from the [vite-plugin-svelte], even though I have set a negative tabIndex value.

```
[vite-plugin-svelte]  /Users/[user]/Projects/evidence/node_modules/.pnpm/svelte-tiny-linked-charts@1.1.2/node_modules/svelte-tiny-linked-charts/src/LinkedChart.svelte:119:6 A11y: noninteractive element cannot have positive tabIndex value
```

**Comment by Mitcheljager** [comment]
That's an annoying one, the warning is false in this case since it does set the tabindex to the correct value, but since it _could_ be a wrong value it displays the warning. I will attempt a fix, but I might just end up ignoring the warning and maybe set up a warning myself instead.

**Comment by archiewood** [comment]
Even suppressing this warning would be a good outcome for us!

**Comment by Mitcheljager** [comment]
This is fixed in 1.1.4 https://github.com/Mitcheljager/svelte-tiny-linked-charts/commit/19b5253fcb66669c1b034479506b391fe239612c
